deps/jerry/tools/profiler/j2cl.js

```diff
@@ -0,0 +1,25 @@
+
+/* Convert jerry CPU profiler data into collapse stack file.
+ * It convert every input line
+ * time,function1(source1:line1:column1),...,functionn(sourcen:linen:columnn)
+ * to line
+ * functionn(sourcen:linen:columnn);...;function1(source1:line1:column1) time
+ * Call graph is functionn->...->function1
```

Comment:
looks like comment has not been terminated.

Reply:
3x. */ added